### PR TITLE
Fixes part of #40: Added bottom padding to Question Player

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes part of #40: Added bottom padding to Question Player

Before this PR more than 2 test cases were failing on espresso in `QuestionPlayerActivityTest` that were fixed by adding the bottom-padding which was also a requirement as per mocks. 

But still 2 test cases are failing because of nested recyclerview for `SelectionInputInteraction`. 

Screenshot 2021-05-25 at 8 18 19 PM

In both these tests we want to scroll to `index=3` item of the `selection_interaction_recycler_view` but the scrolling is disabled because it's an item of `question_recycler_view` and in current code we cannot scroll to this specific position. 

## Output on Pixel 3 (5.56")
Screenshot 2021-06-24 at 12 31 13 AM

## Output on Pixel 3 XL (6.3")
Screenshot 2021-06-24 at 12 28 54 AM

To fix this I have filed an issue #3370

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@BenHenning @anandwana001 PTAL at the description and can you suggest something on how to fix these failing tests?

**Comment by anandwana001** [comment]
I will take a look at this by tomorrow.

**Comment by BenHenning** [comment]
@rt4914 per our discussion earlier in the week, is further input needed from me for this PR?

File: app/src/main/res/layout-land/question_player_fragment.xml [hunk 1]
```diff
@@ -59,6 +59,7 @@
         android:divider="@android:color/transparent"
         android:dividerHeight="8dp"
         android:overScrollMode="never"
+        android:paddingBottom="@dimen/general_screen_padding_bottom"
```
Comment (BenHenning) [inline app/src/main/res/layout-land/question_player_fragment.xml:62]: Why use a general padding here?

**Review by BenHenning — CHANGES_REQUESTED** [review]
Hmm. I'm not keen on changing padding for the purpose of testing since that has real side effects for users. Do you have a before & after to demonstrate the effects?

File: app/src/main/res/layout-land/question_player_fragment.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout-land/question_player_fragment.xml:62]: @BenHenning I have updated the description. 
To be clear, after adding bottom-padding which was anyway needed as per mocks:
Screenshot 2021-06-24 at 12 31 13 AM


But if we run these two failing test cases on Pixel 3XL (basically a large mobile) all tests pass:
Screenshot 2021-06-24 at 12 28 54 AM

**Comment by rt4914** [comment]
@BenHenning 
In `QuestionPlayerActivityLocalTest` test file following test are failing
`testQuestionPlayer_submitTwoWrongAnswers_checkPreviousHeaderCollapsed`
`testQuestionPlayer_submitTwoWrongAnswers_expandResponse_checkPreviousHeaderExpanded`
`testQuestionPlayer_expandCollapseResponse_checkPreviousHeaderCollapsed`

In `testQuestionPlayer_submitTwoWrongAnswers_checkPreviousHeaderCollapsed` we are checking that the `question_player_recycler_view` child-count should be equal to `5` which seems incorrect to me as it should be `6`[question-content-item, previous-response-item (collapsed), submitted-answer-item, feedback-answer-item, interaction-item and button].
But infact, the test error is that we are trying to match count to `5` instead we have only `4` child item. And this error also seems incorrect.

Same applies for other tests too.

## Question Recycler View Item count should be `6` vs `8`

**Review by BenHenning — APPROVED** [review]
Thanks @rt4914. LGTM.

**Pull request merged**